plugin.video.orftvthek/resources/language/English/strings.po

```diff
@@ -0,0 +1,234 @@
+# XBMC Media Center language file
```

Comment:
change this to `plugin.video.orftvthek/resources/language/resource.language.en_gb/strings.po`

Reply:
done with the new commit